### DISCUSSION TITLE
Class loader error thrown from custom exporter

### DISCUSSION THREAD

**Discussion by maheskrishnan** [open-telemetry/opentelemetry-java-instrumentation#2138] [Q&A]
Hello eveyone
anybody seen this error? 

**[BatchSpanProcessor_WorkerThread-1] WARN io.opentelemetry.sdk.trace.export.BatchSpanProcessor$Worker - Exporter threw an Exception org.apache.kafka.common.config.ConfigException: Invalid value org.apache.kafka.common.serialization.ByteArraySerializer for configuration key.serializer: Class org.apache.kafka.common.serialization.ByteArraySerializer could not be found.**

i am using -Dotel.exporter.jar=myexporter.jar option to use my own custom exporter . myexporter.java does contains the org.apache.kafka.common.serialization.StringSerializer. but when running th application getting this error

StackOverFlow Question: https://stackoverflow.com/questions/65946487/class-not-found-error-in-open-telemetry-exporter

Any help is highly appreciated. Thanks

**Comment by gchapim** [comment]
Hi @maheskrishnan I know it's been a long time since you had that issue, but have you managed to solve the problem?
